pages/download.html

```diff
@@ -212,8 +212,10 @@ <h1>{% blocktrans %}Download Now ({{ stable_version }} Stable){% endblocktrans %
   <div class="fullwidth">
     <p>{% blocktrans %}Check out the <a href="{{ stable_release_announcement }}">release announcement</a> for a list of new features.{% endblocktrans %}</p>
     <p>{% blocktrans with previous_link="#previous" %}Although {{ stable_version }} contains many important updates and bugfixes, it's possible some users may experience issues. If you do, you can still get our previous stable version, <a href="{{ previous_link }}">{{ previous_version }}</a>.{% endblocktrans %}</p>
-    <p>{% trans "Mixxx is available for Windows, macOS, and Linux." %}<br/>
-    {% trans "We happily provide Mixxx for free and donations are appreciated!" %}</p>
+    <p>{% trans "Mixxx is available for Windows, macOS, and Linux." %}</p>
+    <h3>{% trans "If you ever see Mixxx being offered for sale, it is an unauthorized distribution." %}<br/>
```

Comment:
This isn't necessarily true. By publishing software under the GPL we explicitly authorize anyone to sell it so long as they provide source code. Most of the time they probably don't provide source code or even a link to the source code, but there could be someone out there selling Mixxx that actually is complying with the GPL. If there isn't now, there could be in the future. How about something like:

Mixxx is available for free for Windows, macOS, and Linux. If you see Mixxx offered for sale on other websites, there is no need to pay for it because you can get it here for free. There is no "pro" version of Mixxx. The free version is theh full version. Donations are appreciated but not required.

Reply:
we can still say that those versions are not "official" though.  Also they shouldn't be using our name

Reply:
updated text